cairo/ethereum/utils/numeric.cairo

```diff
@@ -20,6 +20,7 @@ from cairo_core.maths import (
     felt252_bit_length,
     felt252_to_bits_rev,
     felt252_to_bytes_le,
+    felt252_bytes_length,
```

Comment:
now that we have it it's a good time to remove `test_should_return_bytes_used_in_128_word` (py), `test__bytes_used_128` (cairo) and in https://github.com/kkrt-labs/keth/blob/39cab2eca4f7ab8913e18cad1a41ee0c446de8c6/cairo/ethereum/cancun/vm/instructions/arithmetic.cairo#L549-L560 use this new `felt252_bytes_length` function